test/e2e/resize_nodes.go

```diff
@@ -341,7 +428,8 @@ var _ = Describe("Nodes", func() {
 			}
 		})
 
-		testName = "should survive network partition."
+		// TODO: this test has nothing to do with resizing nodes so it should be moved elsewhere.
+		testName = "should survive unresponsive node."
```

Comment:
At the risk of being pedantic, "Nodes should survive unresponsive node" is not actually what you're testing here.  You're testing that "Pods on uncontactable nodes disappear and are recreated by replication controller"